File: _src_old/main/java/com/hoi4utils/hoi4mod/map/state/Claim.java
package com.hoi4utils.hoi4mod.map.state;

import com.hoi4utils.hoi4mod.common.country_tags.CountryTag;
import com.hoi4utils.parser.ClausewitzDate;

/*
 * Claim File
 */
public record Claim (CountryTag tag, State state, ClausewitzDate date, boolean definedInHistory) {

}

File: src/main/scala/com/hoi4utils/hoi4mod/map/state/Claim.scala
package com.hoi4utils.hoi4mod.map.state

import com.hoi4utils.hoi4mod.common.country_tags.CountryTag
import com.hoi4utils.parser.ClausewitzDate

/**
 * Claim File
 */
case class Claim(tag: CountryTag, state: State, date: ClausewitzDate, definedInHistory: Boolean)